classes/class-wc-emspay-gateway.php

```diff
@@ -119,30 +125,56 @@ public function get_icon()
      */
     protected function ginger_validate_currency() {
 
-        $current_currency = get_woocommerce_currency();
-        if ( ! $this->gingerIsGatewayCurrencySupported() ) {
-            $this->enabled = false;
-            $this->update_option('enabled', false);
+        if(empty($this->ems)) {
+            $reason = __( 'API key is empty. Set API key and try again', WC_Emspay_Helper::DOMAIN );
+            $this->gingerDisabledPaymentMethod($reason);
+            return false;
+        }
+
+        try {
+            $payment_methods_currencies = $this->ems->send('GET', '/merchants/self/projects/self/currencies');
```

Comment:
Джиджер либа обновилась, теперь есть функция getCurrencyList(), но я думаю это я уже изменю, когда начну рефакторинг🙂

Reply:
After i updated the plugin. we cant reach the payment method. Because it say's the API key is not correct. But i changed it a 10 times. Please help me.